modules/loader.js

```diff
@@ -257,7 +257,7 @@ __private.loadTransactions = function (cb) {
 				} catch (e) {
 					library.logger.debug('Transaction normalization failed', {id: id, err: e.toString(), module: 'loader', tx: transaction});
 
-					library.logger.warn(['Transaction', id, 'is not valid, peer removed'].join(' '), peer.string);
+					library.logger.warn(['Transaction', id, 'is not valid, peer removed'].join(' '), peer.ip + ':' + peer.port);
```

Comment:
Why that change? If there is something wrong with `peer.string` - we should fix globally.

Reply:
Agree, keep as it was using `peer.string`